src/main/js/main_html5.js

```diff
@@ -490,6 +490,34 @@ require("../../../html5-common/js/utils/environment.js");
     };
 
     /**
+     * Checks to see if autoplay requires the video to be muted
+     * @public
+     * @method OoyalaVideoWrapper#requiresMutedAutoplay
+     * @param {boolean} true if video must be muted to autoplay, false otherwise
+     */
+    this.requiresMutedAutoplay = function() {
+      return OO.isSafari || OO.isIos || OO.isAndroid;
```

Comment:
Same comment as other PR's regarding the Safari version, looks good otherwise.

Reply:
I'll add that in.